### PR TITLE
Incorporated all of Jasna's edits

### PR DESCRIPTION
**Pull request by pamlane**
Please review changes. I've added comments starting with "pl " for questions from Jasna that I couldn't resolve.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,184 +1,165 @@
...
+Python wrapper for the Streams REST API for monitoring Streams instances and jobs, including accessing the data from views on streams.
+</LI>
+</UL>
+<p>
+See the documentation for details:
+<UL>
+  <li><a target="_blank" href="http://ibmstreams.github.io/streamsx.documentation/docs/python/1.6/python-appapi-devguide/index.html">Developing IBM Streams Applications with Python (Version 1.6)</a></li>
+  <li><a target="_blank" href="https://ibmstreams.github.io/streamsx.topology/doc/releases/1.6/pythondoc/index.html">Python <code>streamsx</code> package documentation</a></li>
+  <li><a target="_blank" href="https://ibmstreams.github.io/streamsx.topology/doc/releases/1.6/spldoc/html/tk$com.ibm.streamsx.topology/tk$com.ibm.streamsx.topology.html">SPL topology toolkit</a></li>
+  <li><a target="_blank" href="http://ibmstreams.github.io/streamsx.topology/doc/releases/1.6/pythondoc/streamsx.spl.spl.html?cm_mc_uid=00312402530514948690745&cm_mc_sid_50200000=1494869074#python-classes-as-spl-operators">Python classes as SPL operators</a></li>
```
Comment (ddebrunner) [inline index.html]: The url should not include additional uid values.

**Comment by pamlane** [comment]
Hi Dan,
 
Since I've had so many git issues with this particular file (the only way I got this to work last time was by doing everything on github directly instead of on my local machine), I made the change directly to the file and committed it. I don't see any option to create another pull request. Hopefully, this is all I need to do and you can go ahead and accept the changes.
 
Thanks,
Pam
 
----- Original message -----From: Dan Debrunner <notifications@github.com>To: "IBMStreams/streamsx.topology" <streamsx.topology@noreply.github.com>Cc: Pam Lane <pamlane@us.ibm.com>, Author <author@noreply.github.com>Subject: Re: [IBMStreams/streamsx.topology] Incorporated all of Jasna's edits (#1133)Date: Fri, Jul 28, 2017 8:59 AM 
@ddebrunner commented on this pull request.
In index.html:
> +<UL>+<LI>Python 2.7 support for Python Application API.</LI>+<LI>Simplified development with Python Application API by supporting lambda functions, callables defined in the main file, and iterables defined directly as source functions.</LI>+<LI>New SPL decorators that 1) create stateful or stateless operators with operator parameters and 2) support passing SPL tuple attributes by name.</LI>+<LI> Python access to the Streams execution context including custom metrics.</LI>+<LI>+Python wrapper for the Streams REST API for monitoring Streams instances and jobs, including accessing the data from views on streams.+</LI>+</UL>+<p>+See the documentation for details:+<UL>+  <li><a target="_blank" href="http://ibmstreams.github.io/streamsx.documentation/docs/python/1.6/python-appapi-devguide/index.html">Developing IBM Streams Applications with Python (Version 1.6)</a></li>+  <li><a target="_blank" href="https://ibmstreams.github.io/streamsx.topology/doc/releases/1.6/pythondoc/index.html">Python <code>streamsx</code> package documentation</a></li>+  <li><a target="_blank" href="https://ibmstreams.github.io/streamsx.topology/doc/releases/1.6/spldoc/html/tk$com.ibm.streamsx.topology/tk$com.ibm.streamsx.topology.html">SPL topology toolkit</a></li>+  <li><a target="_blank" href="http://ibmstreams.github.io/streamsx.topology/doc/releases/1.6/pythondoc/streamsx.spl.spl.html?cm_mc_uid=00312402530514948690745&cm_mc_sid_50200000=1494869074#python-classes-as-spl-operators">Python classes as SPL operators</a></li>
The url should not include additional uid values.

**Comment by ddebrunner** [comment]
FYI - adding commits to the branch adds them to any outstanding pull request.

**Comment by ddebrunner** [comment]
Going to merge this and then remove the embedded questions.

**Pull request merged**